client/packages/invoices/src/OutboundShipment/api/api.ts

```diff
@@ -154,6 +154,20 @@ const outboundParsers = {
 
 export const getOutboundQueries = (sdk: Sdk, storeId: string) => ({
   get: {
+    barcodesByValue: async (
+      value?: string
+    ): Promise<{
```

Comment:
`Promise<BarcodesQuery["barcodes"]>` -> would also mean if we update query we don't need to change things here